tests/com_wrapper_int/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(theseTestsName com_wrapper_int)
+
+set(${theseTestsName}_test_files
+${theseTestsName}.c
+)
+
+set(${theseTestsName}_c_files
+../com_wrapper_ut/test_object.c
+custom_alloc_helpers.c
+test_object_custom_alloc_wrapper.c
+../com_wrapper_ut/test_interface_i.c
+)
+
+set(${theseTestsName}_h_files
+../../inc/com_wrapper/com_wrapper.h
+../com_wrapper_ut/test_object.h
```

Comment:
I think we should duplicate the test_object.h and test_interface.h in the int project in order to make the int project self contained. If anyone makes a change to the UT project it should not indirectly affect the int project.